plugin.php

```diff
@@ -135,6 +135,39 @@
 	return $pDI->get(EstateViewDocumentTitleBuilder::class)->buildDocumentTitle($title);
 }, 10, 2);
 
+if (in_array('wordpress-seo/wp-seo.php', apply_filters('active_plugins', get_option('active_plugins')))) {
```

Comment:
This would only work for this specific plugin. People may choose another plugin and then it would break.

The appraoch with the custom fields works for all SEO plugins that I have found. What was the issue with that approach?

Reply:
Yes, I will try it with another plugin. and research solutions for this issue

Reply:
I tried it with another plugin (ex: Rank Math SEO)
I see that:
Each plugin will have a private hook to custom variables
ex:
Seo Yoast: wpseo_register_extra_replacements
Rank Math SEO: rank_math/vars/register_extra_replacements

I pushed the code and you can see it in this video :
https://www.loom.com/share/ca725ade1d354f7ab9c70b3d931d5cc1

So we can only support this feature for a specific list of plugins.

Please provide plugins, Which you want to implement this feature, or give me your opinion on this isssue.

Reply:
I have found a way to get the estate ID.

For testing, I inserted the following code into my theme's `functions.php`:
```php
function add_oo_meta($value, $object_id, $meta_key) {]
    $detail_page_id = 148;
    if ($object_id == $detail_page_id && $meta_key == "oo_estate_id") {
        return print_r(get_query_var("estate_id"), true);
    } else {
        return null;
    }
}
add_filter("get_post_metadata", "add_oo_meta", 1, 3);
```

I can then add the custom field `oo_estate_id` in Yoast:

![grafik](https://user-images.githubusercontent.com/90763364/163189078-5819b8c6-8b09-4ad3-9ef0-cf20b150e901.png)

This allows me to put the estate ID into the title. In a sense, it goes from the URL to the query var to the custom field to the title.

![grafik](https://user-images.githubusercontent.com/90763364/163189443-6cf201b9-6cd5-4fed-87cf-c81457d0ccf2.png)

If you can access the estate ID in the hook for the custom field, can you then get the desired information about the estate?

Reply:
Yes, I will try it.

Reply:
@jmaas-onoffice I resolved this issue.
Pls, review the code again.
This video test:
https://www.loom.com/share/374bd37519384d2a9ea7240d4b3ea3a8